src/sqlancer/tidb/oracle/TiDBTLPWhereOracle.java

```diff
@@ -42,11 +45,29 @@ public void check() throws SQLException {
                 thirdQueryString, combinedString, !orderBy, state, errors);
         ComparatorHelper.assumeResultSetsAreEqual(resultSet, secondResultSet, originalQueryString, combinedString,
                 state);
+        reproducer = globalState -> {
```

Comment:
Can you give more documents about how to support reproducers for a new DBMS? For this code, I am not very sure how it works, and it seems different from the reproducer in the NoREC oracle.

Reply:
Can you change it to a unified format as the reproducer in NoREC?